tests/client/actions/companiesGetCompany.test.js

```diff
@@ -22,16 +25,16 @@ jest.mock('superagent', () => {
 
 test('get companies gets the correct actions', () => {
   const dispatch = jest.fn()
-  return getCompany('/api/v1/companies')(dispatch)
+  return getCompanyInfo(2)(dispatch)
     .then(() => {
-      expect(dispatch.mock.calls[0][0].type).toBe(RECEIVE_COMPANY)
+      expect(dispatch.mock.calls[0][0].type).toBe(RECEIVE_COMPANY_INFO)
     })
 })
 
 test('get companies returns an error with incorrect url', () => {
   const dispatch = jest.fn()
-  return getCompany('/companies')(dispatch)
-    .catch(() => {
+  return getCompanyInfo('do words')(dispatch)
```

Comment:
replace `do word` with something that makes sense to developers that will read this in the future